src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php

```diff
@@ -48,6 +48,11 @@ final class ClassAttributesSeparationFixer extends AbstractFixer implements Conf
      * @internal
      */
     public const SPACING_ONE = 'one';
+    
+    /**
+     * @internal
+     */
+    public const SPACING_TWO = 'two';
```

Comment:
I am against introducing `SPACING_TWO` because what if somewhere exists coding standard with 3 empty lines? Are we going to add `SPACING_THREE` option and nest more ternaries in the logic?

If we want to support more empty lines, it should be done with generic option, that would allow providing exact number of expected empty lines, so one can use 2, 5 or 10 ("_code likes to breathe_").